README.md

```diff
@@ -105,7 +105,10 @@ sampler.pause()
 sampler.resume()
 
 # Wait for the sampler to finish (up to timeout seconds)
-# sampler.wait(timeout=0.1)
+sampler.wait(timeout=0.1)
+# Note that not passing any timeout to `wait` will
+# directly save and return the results:
```

Comment:
the current results?

Reply:
Yep. Just updated so that's even clearer